cli/types/cypress.d.ts

```diff
@@ -904,13 +904,20 @@ declare namespace Cypress {
     clearCookie(name: string, options?: CookieOptions): Chainable<null>
 
     /**
-     * Clear all browser cookies for the current hostname or for the domain specified.
+     * Clear browser cookies for the current hostname or for the domain specified.
      * Cypress automatically clears all cookies before each test to prevent state from being shared across tests. You shouldn't need to use this command unless you're using it to clear all cookies or specific cookies inside a single test.
      *
      * @see https://on.cypress.io/clearcookies
      */
     clearCookies(options?: CookieOptions): Chainable<null>
 
+    /**
+     * Clear all browser cookies
```

Comment:
```suggestion
     * Clear all browser cookies.
```

Reply:
Addressed in https://github.com/cypress-io/cypress/pull/25012/commits/6c04ba02ab6155c83f8158808d3998d8a0b29f36